charts/dvpe-deployment-gloo/templates/gloo_virtualservice.yaml

```diff
@@ -13,20 +14,40 @@ spec:
     secretRef:
       name: {{ .Values.gloo.virtualservice.spec.sslConfig.secretRef.name }}
       namespace: {{default .Values.gloo.namespace .Values.gloo.virtualservice.spec.sslConfig.secretRef.namespace }}
+    sniDomains:
+      - {{ .Values.gloo.virtualservice.spec.virtualHost.domains }}
   virtualHost:
     domains:
       - {{ .Values.gloo.virtualservice.spec.virtualHost.domains }}
+    {{- if .Values.gloo.virtualservice.spec.virtualHost.cors.allowOrigin }}
+    options:
+      cors:
+        allowCredentials: true
+        allowHeaders:
+          {{- range .Values.gloo.virtualservice.spec.virtualHost.cors.allowHeaders }}
+          - {{ . }}
+          {{- end }}
+        allowMethods:
+          {{- range .Values.gloo.virtualservice.spec.virtualHost.cors.allowMethods }}
+          - {{ . }}
+          {{- end }}
+        allowOrigin:
+          {{- range .Values.gloo.virtualservice.spec.virtualHost.cors.allowOrigin }}
+          - {{ . | quote }}
+          {{- end }}
+        allowOriginRegex:
```

Comment:
Should be removed, if we not use it

Reply:
I'm not sure if this is required by Gloo even if empty - will test it.